csrc/multidevice/communication.h

```diff
@@ -107,6 +108,10 @@ class Communication : public Expr {
     return attribute<int64_t>(4);
   }
 
+  CommunicatorBackend& backend() {
```

Comment:
```suggestion
  CommunicatorBackend backend() {
```
given CommunicatorBackend is merely an enum, an indirection probably isn't worthwhile.